bundles/org.openhab.binding.evcc/README.md

```diff
@@ -38,15 +38,17 @@ Default value for _refreshInterval_ is 60 seconds.
 Please note that some of them are only available when evcc is properly configured.
 
-| Channel                    | Type                 | Read/Write | Description                                                                                                  |
-|----------------------------|----------------------|------------|--------------------------------------------------------------------------------------------------------------|
-| general#batteryCapacity    | Number:Energy        | R          | Capacity of (home) battery.                                                                                  |
-| general#batteryPower       | Number:Power         | R          | Current power from battery.                                                                                  |
-| general#batterySoC         | Number:Dimensionless | R          | Current State of Charge of battery.                                                                          |
-| general#batteryPrioritySoC | Number:Dimensionless | RW         | State of State of Charge for which the battery has priority over charging the ev when charging mode is "pv". |
-| general#gridPower          | Number:Power         | R          | Current power from grid (negative means feed-in)                                                             |
-| general#homePower          | Number:Power         | R          | Current power taken by home.                                                                                 |
-| general#pvPower            | Number:Power         | R          | Current power from photovoltaik.                                                                             |
+| Channel                         | Type                 | Read/Write | Description                                                                                                  |
+|---------------------------------|----------------------|------------|--------------------------------------------------------------------------------------------------------------|
+| general#batteryCapacity         | Number:Energy        | R          | Capacity of (home) battery                                                                                   |
+| general#batteryPower            | Number:Power         | R          | Current power from battery                                                                                   |
+| general#batterySoC              | Number:Dimensionless | R          | Current State of Charge of battery                                                                           |
+| general#batteryPrioritySoC      | Number:Dimensionless | RW         | State of State of Charge for which the battery has priority over charging the ev when charging mode is "pv"  |
+| general#batteryDischargeControl | Switch               | RW         | Battery Discharge Control switch                                                                             |
```

Comment:
```suggestion
| general#batteryDischargeControl | Switch               | RW         | Battery Discharge Control                                                                                    |
```

Reply:
took over the same description as in `thing-types.xml`